src/libAtomVM/CMakeLists.txt

```diff
@@ -89,6 +93,16 @@ gperf_generate(${CMAKE_CURRENT_SOURCE_DIR}/nifs.gperf nifs_hash.h)
 add_custom_target(generated DEPENDS bifs_hash.h)
 add_custom_target(generated-nifs-hash DEPENDS nifs_hash.h)
 
+execute_process(
+    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+    COMMAND git rev-parse HEAD
```

Comment:
Here we are making the assumption that the build system has git, also we are coupling the build system with the version control system. Users might build AtomVM from a tar.bz or a zip archive from github, so it would break.
I would rather manually bump the version (or use a build date in the worst case).

Reply:
In that case, in the absence of a release procedure, we should remove the field.  I'd rather not have the field, if it can't be managed.  Otherwise it will just float out of date.

Reply:
Removed.